collect_app/src/main/java/org/odk/collect/android/geo/OsmDroidMapFragment.java

```diff
@@ -175,12 +175,6 @@ public void onStop() {
         super.onStop();
     }
 
-    @Override
-    public void onDestroy() {
```

Comment:
I had to remove that because it caused NPEs in OSM I don;t think it's needed, at least I'm not seeing any leaks.